lighthouse-core/lib/network-request.js

```diff
@@ -103,6 +103,18 @@ module.exports = class NetworkRequest {
     this.fetchedViaServiceWorker = false;
     /** @type {string|undefined} */
     this.frameId = '';
+    /**
+     * @type {string|undefined}
+     * Only set for OOPIFs. This is the targetId of the protocol target from which this
```

Comment:
technically according to the types this can be undefined even when there was a source, but I have never been able to see this happen with an OOPIF